Xamarin.Forms.Maps.Android/MapRenderer.cs

```diff
@@ -10,311 +11,358 @@
 
 			var incc = Map.Pins as INotifyCollectionChanged;
 			if (incc != null)
+			{
 				incc.CollectionChanged += OnCollectionChanged;
-		}
-
-		void OnCollectionChanged (object sender, NotifyCollectionChangedEventArgs notifyCollectionChangedEventArgs)
-		{
-			switch (notifyCollectionChangedEventArgs.Action) {
-			case NotifyCollectionChangedAction.Add:
-				AddPins (notifyCollectionChangedEventArgs.NewItems);
-				break;
-			case NotifyCollectionChangedAction.Remove:
-				RemovePins (notifyCollectionChangedEventArgs.OldItems);
-				break;
-			case NotifyCollectionChangedAction.Replace:
-				RemovePins (notifyCollectionChangedEventArgs.OldItems);
-				AddPins (notifyCollectionChangedEventArgs.NewItems);
-				break;
-			case NotifyCollectionChangedAction.Reset:
-					_markers?.ForEach (m => m.Remove ());
-					_markers = null;
-				AddPins ((IList)(Element as Map).Pins);
-				break;
-			case NotifyCollectionChangedAction.Move:
-				//do nothing
-				break;
-			}
-		}
-
-		void OnMoveToRegionMessage (Map s, MapSpan a)
-		{
-			MoveToRegion (a, true);
-		}
-
-		void MoveToRegion (MapSpan span, bool animate)
-		{
-			var map = NativeMap;
-			if (map == null)
-				return;
-
-			span = span.ClampLatitude (85, -85);
-			var ne = new LatLng (span.Center.Latitude + span.LatitudeDegrees / 2, span.Center.Longitude + span.LongitudeDegrees / 2);
-			var sw = new LatLng (span.Center.Latitude - span.LatitudeDegrees / 2, span.Center.Longitude - span.LongitudeDegrees / 2);
-			var update = CameraUpdateFactory.NewLatLngBounds (new LatLngBounds (sw, ne), 0);
-	
-			try {
-				if (animate)
-					map.AnimateCamera (update);
-				else
-					map.MoveCamera (update);
-			} catch (IllegalStateException exc) {
-				System.Diagnostics.Debug.WriteLine ("MoveToRegion exception: " + exc);
-			}
-		}
-
-		bool _init = true;
-
-		protected override void OnLayout (bool changed, int l, int t, int r, int b)
-		{
-			base.OnLayout (changed, l, t, r, b);
-
-			if (_init) {
-				MoveToRegion (((Map)Element).LastMoveToRegion, false);
-				OnCollectionChanged (((Map)Element).Pins, new NotifyCollectionChangedEventArgs( NotifyCollectionChangedAction.Reset));
-				_init = false;
-			} else if (changed) {
-				UpdateVisibleRegion (NativeMap.CameraPosition.Target);
 			}
 		}
 
-		protected override void OnElementPropertyChanged (object sender, PropertyChangedEventArgs e)
+		protected override void OnElementPropertyChanged(object sender, PropertyChangedEventArgs e)
 		{
-			base.OnElementPropertyChanged (sender, e);
+			base.OnElementPropertyChanged(sender, e);
 
-			if (e.PropertyName == Map.MapTypeProperty.PropertyName) {
+			if (e.PropertyName == Map.MapTypeProperty.PropertyName)
+			{
 				SetMapType();
 				return;
 			}
-			
-			var gmap = NativeMap;
+
+			GoogleMap gmap = NativeMap;
 			if (gmap == null)
+			{
 				return;
+			}
 
 			if (e.PropertyName == Map.IsShowingUserProperty.PropertyName)
+			{
```

Comment:
... and, looking at the our [code style guidelines](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md)  it looks like the corefx rule is that [if any block in the statement needs braces, they all do](https://github.com/dotnet/corefx/issues/381#issuecomment-69474542).